lib/dish.rb

```diff
@@ -0,0 +1,14 @@
+class Dish
+  attr_reader :name, :price, :hash
+
+  def initialize(name = "Hash brown", price = 0.99)
+    @name = name
+    @price = price
+    @hash = {}
+  end
+
+  def itemise
+    @hash[@name] = @price.round(2)
```

Comment:
This .round method is clever, I ended up using integers as I found working with floats difficult.